block/manager_test.go

```diff
@@ -404,12 +403,78 @@ func TestBatchRetryWhileBatchAccepted(t *testing.T) {
 	manager.pubsub.PublishWithEvents(ctx, eventData, map[string][]string{settlement.EventTypeKey: {settlement.EventNewSettlementBatchAccepted}})
 	time.Sleep(200 * time.Millisecond)
 	// Change settlement layer to accept batches and validate new batches was submitted
-	settlementLayerClient.isError.Store(false)
+	settlementLayerClient.IsError.Store(false)
 	blockLoopContext, blockLoopCancel = context.WithCancel(context.Background())
 	defer blockLoopCancel()
 	go manager.ProduceBlockLoop(blockLoopContext)
 	time.Sleep(1 * time.Second)
-	assert.Greater(atomic.LoadUint64(&settlementLayerClient.batchCounter), uint64(0))
+	assert.Greater(atomic.LoadUint64(&settlementLayerClient.BatchCounter), uint64(0))
+
+}
+
+func Test_createNextDABatchWithBytesLimit(t *testing.T) {
```

Comment:
So far our testing convention in dymint is pascal case. 
not sure if the underscore is by mistake or a different convention by I rather we keep it unified and use pascal case.